blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServerBuilder.scala

```diff
@@ -106,7 +109,8 @@ class BlazeServerBuilder[F[_]](
       bufferSize: Int = bufferSize,
       selectorThreadFactory: ThreadFactory = selectorThreadFactory,
       enableWebSockets: Boolean = enableWebSockets,
-      sslBits: Option[SSLConfig] = sslBits,
+      sslContext: F[Option[SSLContext]] = sslContext,
```

Comment:
`withSSL`'s old, quirky `KeyStoreBits` is effectful in creating the SSLContext, so we need to wrap this in an effect.